common/core/desktop/tests/unit/kmx/042 - long context and split deadkeys.kmn

```diff
@@ -1,14 +1,23 @@
 c Description: Tests that we don't split a dk in context. Note that we start with a context with length MAXCONTEXT.
```

Comment:
This has the same issue as test 041 I think?

Reply:
yep